npm-publish/bin/publish.sh

```diff
@@ -16,7 +16,7 @@ echo_title() {
 # Determine which files were changed in PR
 echo_title "Determining which files were changed in PR #$PR_NUMBER"
 set +o errexit # temporary do not exit on error because grep will exit with error when nothing is found
-PR_FILES_CHANGED=`gh pr diff "$PR_NUMBER" --name-only | grep -v '\.json' | wc -l`
+PR_FILES_CHANGED=$(gh pr diff "$PR_NUMBER" --name-only | grep -vFc '.json')
```

Comment:
[SC2126](https://www.shellcheck.net/wiki/SC2126): Consider using `grep -c` instead of `grep | wc`